taxi/forms.py

```diff
@@ -0,0 +1,61 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django.core.validators import MaxLengthValidator, MinLengthValidator
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",)
+
+    def clean_license_number(self):
+        license_num = self.cleaned_data["license_number"]
+        if not license_num[:3].isupper() or not license_num[:3].isalpha():
+            raise ValidationError(
+                "First 3 characters must be uppercase letters."
+            )
+
+        if not license_num[3:].isdigit():
+            raise ValidationError("Last 5 characters must be digits.")
+
+        return license_num
```

Comment:
you can make a separate function for it or mixin because you use the same code twice